jenkins-pipelines/build-dex.Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+import java.text.SimpleDateFormat
+
+class Globals {
+    static String dockerTag = null
+    static String dockerCacheArg = null
+}
+
+def legion
+
+pipeline {
+    agent any
+    options{
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        disableConcurrentBuilds()
+    }
+    environment {
+        build_workspace = "${WORKSPACE}"
+        def dex_dockerimage = "k8s-dex"
+        shared_lib_path = "deploy/legionPipeline.groovy"
+        //parameters section
+        //git branch to checkout 
+        //Set main branch for Job SCM GitBranch
+        //git branch to checkout legion
+        git_branch_legion = "${params.GitBranchLegion}"
```

Comment:
can we use camelCase variables naming which is common groovy variables naming convention

Reply:
Update variables names in pipeline